### DISCUSSION TITLE
Found 2 elements with non-unique id #rc_select_0

### DISCUSSION THREAD

**Discussion by ghost** [ant-design/ant-design#28607] [Q&A] [answered]
Hi

I have a problem in my project as shown on below screenshot

![ZCcP3](https://user-images.githubusercontent.com/76407133/103300938-90dd6b80-4a19-11eb-9fd3-0fc70cd3c5e2.png)

but I can't reproduce it easily on separate project. Can someone please help why this maybe happening?

There are few `Select`s and few `TreeSelect`s on the page where this happens.

antd:4.8.4

I opened an issue after I managed to reproduce, but still I am not getting proper help on the issue, so will post link here
https://github.com/ant-design/ant-design/issues/28609

**Comment by afc163** [comment, marked as answer]
https://github.com/ant-design/ant-design/issues/28609
